source/features/link-to-compare-diff.tsx

```diff
@@ -0,0 +1,26 @@
+import './link-to-compare-diff.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {wrapAll} from '../helpers/dom-utils';
+
+function init(): void {
+	const changedFilesSummary = select('.Box .octicon-file-diff')!.closest('li')!;
+	wrapAll(
+		[...changedFilesSummary.children],
+		<a className="no-underline rgh-link-to-compare-diff" href="#diff"/>,
```

Comment:
`#diff` link does not work on new tab, `#files_bucket` does:

```suggestion
		<a className="no-underline rgh-link-to-compare-diff" href="#files_bucket"/>,
```